src/psbt/finalizer.rs

```diff
@@ -351,7 +352,9 @@ fn interpreter_inp_check<C: secp256k1::Verification, T: Borrow<TxOut>>(
     Ok(())
 }
 
-/// Finalize the psbt. This function takes in a mutable reference to psbt
+/// Finalize the psbt.
```

Comment:
Maybe
```suggestion
/// Finalizes the psbt.
```

to adhere with other docstrings?